bin/README.md

```diff
@@ -0,0 +1,14 @@
+This folder contains scripts for performing various tasks related to the MARTI dice server.
+
+## Create and initialize the staging database
+
+1. Login to `tripleawarclub.org`.
```

Comment:
Probably forgot to mention this on other occurrences but it would be better if we'd use dice.tripleawarclub.org as the login url as I plan to create a CNAME entry to link to forums.triplea-game.org which makes you try to login to the false server instead of configuring the server to perform 301 redirects.
Not sure how well this is going to work out though so I might need to change the strategy. 
In any case we're better off changing it just to be sure.

Reply:
Good point.  I'll update the hostname.

Reply:
Done in rebased d272c5d.